images/win/toolset-2016.json

```diff
@@ -47,6 +47,26 @@
                 "12.*",
                 "14.*"
             ]
+        },
+        {
+            "name": "Boost",
+            "url" : "https://raw.githubusercontent.com/akv-platform/boost-hostedtoolcache/master/versions-manifest.json",
+            "arch": "x86_64",
+            "platform" : "win32",
+            "toolset": "msvc14.1",
+            "versions": [
+                "1.69.0"
+            ]
+        },
+        {
+            "name": "Boost",
+            "url" : "https://raw.githubusercontent.com/akv-platform/boost-hostedtoolcache/master/versions-manifest.json",
+            "arch": "x86_64",
+            "platform" : "win32",
+            "toolset": "msvc14.1",
+            "versions": [
+                "1.71.0"
+            ]

```

Comment:
Why we can't merge both objects to the single one?
`versions` is array and you can put more than 1 version here. See Python as example

Reply:
Thank you, fixed.